CondFormats/PCLConfig/plugins/AlignPCLThresholdsReader.cc

```diff
@@ -0,0 +1,128 @@
+#include <string>
+#include <iostream>
+#include <map>
+#include "FWCore/Framework/interface/EDAnalyzer.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "CondFormats/PCLConfig/interface/AlignPCLThresholds.h"
+#include "CondFormats/DataRecord/interface/AlignPCLThresholdsRcd.h"
+
+namespace edmtest
+{
+  class AlignPCLThresholdsReader : public edm::EDAnalyzer
+  {
+  public:
+    explicit  AlignPCLThresholdsReader(edm::ParameterSet const& p) :
+      printdebug_(p.getUntrackedParameter<bool>("printDebug",true)),
+      formatedOutput_(p.getUntrackedParameter<std::string>("outputFile",""))
+    { 
+      edm::LogInfo("AlignPCLThresholdsReader")<<"AlignPCLThresholdsReader"<<std::endl;
+    }
+    explicit  AlignPCLThresholdsReader(int i) {
+      edm::LogInfo("AlignPCLThresholdsReader")<<"AlignPCLThresholdsReader "<<i<<std::endl; 
+    }
+    virtual ~AlignPCLThresholdsReader() {  
+      edm::LogInfo("AlignPCLThresholdsReader")<<"~AlignPCLThresholdsReader "<<std::endl;
+    }
+    virtual void analyze(const edm::Event& e, const edm::EventSetup& c) override;
+
+  private:
+    bool printdebug_;
+    std::string formatedOutput_;
```

Comment:
I believe these could be both `const` as is in general good practice for configuration parameters